src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -569,6 +569,8 @@ override public void Prepare()
 
 
             SqlStatistics statistics = null;
+            try
+            {
```

Comment:
Fix the indentation in the try block and the finally block that has been added.

Reply:
Whoa, i'm sure it wasn't like that when I added it... nevermind, fixed.